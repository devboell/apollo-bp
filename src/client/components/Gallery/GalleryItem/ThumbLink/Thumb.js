import styled from 'styled-components'

const Thumb = styled.img`
  max-width: 200px;
  max-height: 200px;
`

export default Thumb
