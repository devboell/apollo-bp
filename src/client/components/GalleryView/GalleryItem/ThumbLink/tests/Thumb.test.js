import React from 'react'
import 'jest-styled-components'

import Thumb from '../Thumb'

it('Thumb, styled component snapshot', () => {
  const wrapper = shallow(<Thumb />)
  expect(wrapper).toMatchStyledComponentsSnapshot()
})
