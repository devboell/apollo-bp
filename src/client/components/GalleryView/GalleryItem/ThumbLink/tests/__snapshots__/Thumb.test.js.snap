// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thumb, styled component snapshot 1`] = `
.c0 {
  max-width: 200px;
  max-height: 200px;
}

<img
  className="c0"
/>
`;
